robot/Cumulus/resources/LevelsPageObject.py

```diff
@@ -91,7 +89,14 @@ def enter_level_dd_values(self, **kwargs):
                 self.selenium.set_focus_to_element(locator)
                 self.selenium.select_from_list_by_label(loc,value)
                 self.selenium.wait_until_element_is_not_visible(spinner)
-    
-        
-        
-    
\ No newline at end of file
+
+@pageobject("Details", "Level__c")
+class LevelDetailPage(BaseNPSPPage, DetailPage): 
+
+    def _is_current_page(self):
+        """ Verify we are on the Level detail page
+            by verifying that the url contains '/view'
+        """
+        self.selenium.wait_until_location_contains("/view", timeout=60, message="Detail view did not open in 1 min")
```

Comment:
Same comment as above for timeouts

Reply:
This timeout is added as part of fixing failing page load issues on MetaCI and was the for all page objects.